packages/relay-service/src/index.js

```diff
@@ -39,6 +39,30 @@ const sendMessage = async ({keyPairDoc, recipientDid, message, type}) => {
     keyAgreementKey,
   });
 
+  const didDocument = await blockchainService.resolveDID(recipientDid);
+  const service = didDocument.service.find(
+    endpoint => endpoint.type === 'DIDCommMessaging',
+  );
+
+  const serviceEndpoint = service?.serviceEndpoint[0];
+
+  // if DIDCommMessaging endpoint is found, use it to send the message
+  if (serviceEndpoint) {
+    try {
+      const result = await axios.post(serviceEndpoint.uri, jweMessage, {
+        headers: {
+          'Content-Type': 'application/didcomm-encrypted+json',
+          Accept: 'application/didcomm-encrypted+json',
+        },
+      });
+
+      return result.data;
```

Comment:
returning result.data here and again below will produce two different results for success. should ideally return the same object or  just a success value if its not actually needed